File: C/exercicio5.c
#include <stdio.h>

int main(void)
{
  float metros = 1, centimetros;
  printf("Digite um valor para ser convertido de metros para centímetros: ");
  do
    {
      scanf("%f", &metros);
      if (metros < 1)
      {
        printf("Digite um valor válido!\n");
      }
    }
  while (metros < 1);
  centimetros = metros * 100;
  printf("O valor %.2f é, em centímetros: %.2f\n", metros, centimetros);
  return 0;
}

File: Python/exercicio5.py
if __name__ == '__main__':
  print("Digite um valor em metros para ser convertido em centímetros:")
  valor = float(input())
  if valor > 0:
    valor *= 100
    print(f"O valor digitado em centímetros é {valor}")
  else:
    print("Digite um valor válido!")
    